superdesk/celery_app.py

```diff
@@ -77,7 +77,7 @@ def handle_exception(exc):
     logger.exception(exc)
 
 
-class AppContextTask(TaskBase):
+class AppContextTask(TaskBase): # type: ignore
```

Comment:
why those `type: ignore` are needed everywhere? Is it something for mypy of something else? It's making the source harder to read. Typing in python being optional and because it can be done progressively, I wonder why this is needed at all.

Reply:
In this case, mypy was complaining that `TaskBase` can not be a base class. 
Mypy does not only type checking. I did not find a configuration to ignore this type of check.

Reply:
That's because `Celery.Task` is actually a function not a Class.

Reply:
There are some unclear moments for me with dynamic types in mypy. 
Basically `Celery.Task` is a property which returns a base class and mypy complains about it (_error: Variable "superdesk.celery_app.TaskBase" is not valid as a type_).
I've tried to define a type for `TaskBase`:

```
TaskBase: Type[celery.Task] = celery.Task
```

but mypy has another opinion:

```
Name 'celery.Task' is not defined
```

Anyway, I've decided to ignore this one for now :)